viam-cartographer_test.go

```diff
@@ -213,6 +213,20 @@ func TestEndpointFailures(t *testing.T) {
 	test.That(t, frame, test.ShouldBeEmpty)
 	test.That(t, fmt.Sprint(err), test.ShouldContainSubstring, "error getting SLAM position")
 
+	callbackPointCloudStream, err := svc.GetPointCloudMapStream(context.Background(), "hi")
```

Comment:
should these be `GetPointCloudMap` and `GetInternalState`?

Reply:
yes, and while I thought I did change them it turns out the slam service interface does not define GetPointCloudMap and GetInternalState yet, so I cannot change these currently

Reply:
after discussing offline, these tests have to be temporarily removed in order to not break the slam service interface